entry/entry.go

```diff
@@ -51,6 +51,7 @@ type Entry struct {
 	TraceId      []byte            `json:"trace_id,omitempty"      yaml:"trace_id,omitempty"`
 	TraceFlags   []byte            `json:"trace_flags,omitempty"   yaml:"trace_flags,omitempty"`
 	Severity     Severity          `json:"severity"                yaml:"severity"`
+	LoggerName   string            `json:"logger_name"             yaml:"logger_name"`
```

Comment:
Do we want this to be called LoggerName or ScopeName? It is called InstrumentationScope Name in the data model.

Reply:
This raises a question for me. In the context of logs, what value(s), if any, would reasonably be used as the scope, other than a logger name? Perhaps the log file name or similar? We could consider each input operator separately. 

If in the context of Logs, logger name is really the only sensible scope, then I would prefer to leave it as LoggerName to encourage use of only that specific value.

I need to think on this more, but at least in the case of file name, I don't think this is really the right value, as it really is a medium and not a scope from which the log was emitted. Perhaps some of the more native inputs like journald, windows event log have a sensible default though.